10_Code/Dosage.py

```diff
@@ -0,0 +1,68 @@
+"""Read and Handle the raw data, subset the data, calculate the dosage, and transform data"""
```

Comment:
While the individual codes are well commented, it would be helpful if this comment gave a bit more description as to why we are doing this code and what we hope to achieve